packages/dx-chart-core/src/types/chart-core.types.ts

```diff
@@ -4,137 +4,141 @@ export type DataItems = ReadonlyArray<DataItem>;
+  /** The point's x coordinate */
   readonly x: number;
-  // The point's y coordinate'
+  /** The point's y coordinate' */
   readonly y: number;
   // Let's keep it here (instead of creating a separate interface with single field) for now.
-  // The point's y1 coordinate
+  /** The point's y1 coordinate */
   readonly y1?: number;
 }
 
-export interface BarPoint extends TransformedPoint {
-  // The bar's width in relative units
-  readonly barWidth: number;
-  // The maximum width that the bar can occupy, measured in pixels
-  readonly maxBarWidth: number;
-}
-
-export interface ScatterPoint extends TransformedPoint {
-  // Point options
-  readonly point: { size: number };
-}
-
-export interface PiePoint extends TransformedPoint {
-  // The slice's maximum radius in pixels
-  readonly maxRadius: number;
-  // The inner radius in relative units
-  readonly innerRadius: number;
-  // The outer radius in relative units
-  readonly outerRadius: number;
-  // The slice's start angle
-  readonly startAngle: number;
-  // The slice's end angle
-  readonly endAngle: number;
-}
-
-export interface Target {
-  // Series name
+/** The object that points at a clicked series */
+export interface SeriesRef {
+  /** Series name */
   readonly series: string;
-  // The point’s index within the data array
+  /** The point’s index within the data array */
   readonly point: number;
 }
-export type TargetList = ReadonlyArray<Target>;
+export type TargetList = ReadonlyArray<SeriesRef>;
 
 export type GetFormatFn = (tick: any) => string;
 
+// TODO: The issue is similar to that of *TransformedPoint*.
+/** @internal */
 export interface Series {
-  readonly index: number;
-  readonly scaleName: string;
-  readonly symbolName: string;
+  /** A series name */
   readonly name: string;
+  /** The name of a data field that provides series point values */
+  readonly valueField: string;
+  /** The name of a data field that provides series point argument values */
+  readonly argumentField: string;
+  /** The associated scale */
+  readonly scaleName?: string;
+  /** A series color */
+  readonly color: string;
+  /** @internal */
+  readonly seriesComponent?: React.ComponentType<any>;
+  /** @internal */
+  readonly pointComponent?: React.ComponentType<any>;
+  /** @internal */
+  readonly index: number;
+  /** @internal */
   readonly getValueDomain?: GetValueDomainFn;
+  /** @internal */
   readonly points: PointList;
-  readonly color: string;
-  readonly argumentField: string;
-  readonly valueField: string;
+  /** @internal */
   readonly createHitTester: CreateHitTesterFn;
+  /** @internal */
   readonly getPointTransformer: GetPointTransformerFn;
+  /** @internal */
+  readonly state?: string;
+  /** @internal */
+  readonly symbolName: unique symbol;
 }
+/** @internal */
 export type SeriesList = ReadonlyArray<Series>;
 
+/** @internal */
 export type GetValueDomainFn = (points: PointList) => DomainItems;
 
 export type Palette = ReadonlyArray<string>;
```

Comment:
Palette is also a plugin, we should export that plugin and rename this type

Reply:
Done. Palette -> Colors